[2022-01-25 10:21:06,561] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: classification.apply_model 2022-01-25T10:20:47.998539+00:00 [queued]>
[2022-01-25 10:21:06,577] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: classification.apply_model 2022-01-25T10:20:47.998539+00:00 [queued]>
[2022-01-25 10:21:06,578] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2022-01-25 10:21:06,578] {taskinstance.py:1043} INFO - Starting attempt 1 of 2
[2022-01-25 10:21:06,578] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2022-01-25 10:21:06,596] {taskinstance.py:1063} INFO - Executing <Task(PythonOperator): apply_model> on 2022-01-25T10:20:47.998539+00:00
[2022-01-25 10:21:06,603] {standard_task_runner.py:52} INFO - Started process 8215 to run task
[2022-01-25 10:21:06,609] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'classification', 'apply_model', '2022-01-25T10:20:47.998539+00:00', '--job-id', '59', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/classification.py', '--cfg-path', '/tmp/tmpu6nlth2l', '--error-file', '/tmp/tmptqls4rp5']
[2022-01-25 10:21:06,614] {standard_task_runner.py:77} INFO - Job 59: Subtask apply_model
[2022-01-25 10:21:06,667] {logging_mixin.py:104} INFO - Running <TaskInstance: classification.apply_model 2022-01-25T10:20:47.998539+00:00 [running]> on host 2dc98551bc76
[2022-01-25 10:21:06,712] {taskinstance.py:1257} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=classification
AIRFLOW_CTX_TASK_ID=apply_model
AIRFLOW_CTX_EXECUTION_DATE=2022-01-25T10:20:47.998539+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-01-25T10:20:47.998539+00:00
[2022-01-25 10:21:20,691] {python.py:118} INFO - Done. Returned value was: None
[2022-01-25 10:21:20,732] {taskinstance.py:1166} INFO - Marking task as SUCCESS. dag_id=classification, task_id=apply_model, execution_date=20220125T102047, start_date=20220125T102106, end_date=20220125T102120
[2022-01-25 10:21:20,884] {taskinstance.py:1220} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2022-01-25 10:21:20,974] {local_task_job.py:146} INFO - Task exited with return code 0
